Ajoute l'article pour annoncer Sud Web 2019 (#56)
sudweb/blog: _posts/2019-01-15-sud-web-2019-cest-reparti.md

---
title: "Sud Web 2019, c’est reparti !"
category: '2019'
locale: fr_FR
description: 'Sud Web revient avec une nouvelle équipe et de nouvelles idées, on vous explique tout.'
author: Sud Web
---

> Vous êtes peut-être reparti·es de la dernière édition Sud Web organisée à Anduze avec de nouvelles inspirations en tête. Depuis, une nouvelle équipe s’est formée et nous n’avons pas donné beaucoup de nouvelles. Maintenant que nous avons suffisamment avancé·es, il est temps de vous expliquer où nous allons en 2019. Direction Toulouse !

Reviewer: ```suggestion
Vous êtes peut-être reparti·e·s de la dernière édition Sud Web organisée à Anduze avec de nouvelles inspirations en tête. Depuis, une nouvelle équipe s’est formée et nous n’avons pas donné beaucoup de nouvelles. Maintenant que nous avons suffisamment avancé, il est temps de vous expliquer où nous allons en 2019. Direction Toulouse !
```

## Une nouvelle équipe organisatrice, de nouvelles envies

> Cette année, on se retrouve à quatre dans l’équipe : deux têtes connues (Bertrand et Alexandra) et deux nouvelles têtes (Tak et Marien). Tout d’abord on s’est demandé : « C’est suffisant, quatre, pour organiser un évènement comme Sud Web ? ». Puis très vite Bertrand nous a rassuré sur le fait que ça s’était déjà fait, sans compter Thomas qui nous ~~torture~~ accompagne pour nous aider à faire émerger l’édition de nos rêves.

Reviewer: ```suggestion
Cette année, on se retrouve à quatre dans l’équipe : deux têtes connues (Bertrand et Alexandra) et deux nouvelles têtes (Tak et Marien). Tout d’abord, on s’est demandé : « C’est suffisant, quatre, pour organiser un évènement comme Sud Web ? ». Puis, très vite, Bertrand nous a rassuré sur le fait que ça s’était déjà fait, sans compter Thomas qui nous ~~torture~~ accompagne pour nous aider à faire émerger l’édition de nos rêves.
```

Et puis Sud Web c’est avant tout une conférence à taille humaine, quatre personnes pour préparer un évènement pour une centaine de personnes, c’est déjà pas mal. Ça nous laisse l’occasion d’expérimenter des choses et de nous faire vraiment plaisir. Le mot d’ordre cette année : on met notre énergie là où on en a envie.

Nos envies d’ailleurs, quelles sont-elles ?

> Pour ma part, mes envies portent sur deux points en particulier : la migration des outils/services utilisés par Sud Web vers des hébergeurs plus locaux et proches de nous, ainsi que de travailler sur l’accueil des participants et participantes pour que chacun·e trouve sa place le plus rapidement possible.
-- Marien

> > Sud Web a toujours été pour moi un lieu de partage et de rencontres, sources d'inspiration. Pour l'année 2019, je souhaite poursuivre dans cette direction et proposer aux participants un nouvel espace dédié à la transmission de nos savoirs techniques et humains.

Reviewer: ```suggestion
> Sud Web a toujours été pour moi un lieu de partage et de rencontres, source d'inspiration. Pour l'année 2019, je souhaite poursuivre dans cette direction et proposer aux participants un nouvel espace dédié à la transmission de nos savoirs techniques et humains.
```

-- Tak Castel

> > Proposer un autre espace de discussion, chercher du sens, ça c’est le cadre général. Pour cette édition, je crois qu'on cherche à mettre en place un côté plus axé sur la pratique. Des retours d’expérience encore, mais plus de démonstratif, plus de possibilité de faire.

Reviewer: > plus de démonstratif, plus de possibilité de faire

J'aurais plutôt utiliser "davantage". En première lecture, ça m'a perturbé, j'avais compris l'inverse du sens voulu :)

-- Bertrand Keller

> > Ce que j'aimerais pour le prochain Sud Web ce serait revoir l'organisation de la journée pour laisser plus de place au forum ouvert. Au niveau du contenu, je verrais bien plus de sujets techniques et concrets autour du web.

Reviewer: ```suggestion
> Ce que j'aimerais pour le prochain Sud Web ce serait revoir l'organisation de la journée pour laisser plus de place au forum ouvert. Au niveau du contenu, je verrais bien plus de sujets techniques et concrets autour du Web.
```

-- Alexandra

## Une organisation organique

Comme on le disait plus haut, on a envie d’essayer de nouvelles choses cette année, on a d’ailleurs commencé à apporter quelques petits changements.

> Le premier, c’est qu’**on a décidé de vous ouvrir notre cuisine interne**. Nos canaux de discussion sur Slack sont accessibles pour observer et dialoguer avec la Thym (les canaux commencent tous par `orga-`). Il faudra [commencer par vous inscrire](http://sudweb.herokuapp.com/). Si vous voulez donner un coup de main, vous êtes les bienvenu·es. Nos décisions seront, on l’espère, plus transparentes : aucun tabou ! C’est une démarche expérimentale, pour un travail bénévole sur notre temps libre, nous vous encourageons donc à en tenir compte dans vos propositions.

Reviewer: ```suggestion
Le premier, c’est qu’**on a décidé de vous ouvrir notre cuisine interne**. Nos canaux de discussion sur Slack sont accessibles pour observer et dialoguer avec la Thym (les canaux commencent tous par `orga-`). Il faudra [commencer par vous inscrire](http://sudweb.herokuapp.com/). Si vous voulez donner un coup de main, vous êtes les bienvenu·e·s. Nos décisions seront, on l’espère, plus transparentes : aucun tabou ! C’est une démarche expérimentale, pour un travail bénévole sur notre temps libre, nous vous encourageons donc à en tenir compte dans vos propositions.
```

> Le deuxième changement, c’est qu’on s’est dit que c’était l’occasion de revoir notre utilisation des différents outils-services que nous utilisions. Comme Sud Web est un lieu de partage, de lien humain et où l’on fait la part belle à la critique des usages du web, c’était assez ironique de nous enfermer chez les GAFAM. Et puis comme Marien a rejoint l’équipe justement pour mener cette réflexion, ça aurait été dommage de s’en priver ! Il y aurait théoriquement beaucoup de choses à migrer alors on ira à notre rythme, en commençant par déplacer [les dépôts GitHub](https://github.com/sudweb/) vers [Framagit](https://framagit.org/sudweb).

Reviewer: ```suggestion
Le deuxième changement, c’est qu’on s’est dit que c’était l’occasion de revoir notre utilisation des différents outils/services que nous utilisions. Comme Sud Web est un lieu de partage, de lien humain et où l’on fait la part belle à la critique des usages du Web, c’était assez ironique de nous enfermer chez les <abbr title="Google, Apple, Facebook, Amazon et Microsoft">GAFAM</abbr>. Et puis comme Marien a rejoint l’équipe justement pour mener cette réflexion, ça aurait été dommage de s’en priver ! Il y aurait théoriquement beaucoup de choses à migrer alors on ira à notre rythme, en commençant par déplacer [les dépôts Github](https://github.com/sudweb/) vers [Framagit](https://framagit.org/sudweb).
```

Deux changements plutôt importants pour nous donc, mais peut-être pas les derniers. On continue de cogiter de notre côté pour faire correspondre l’édition 2019 avec nos envies, et si cela nécessite de faire encore évoluer notre organisation, on ne se privera pas.

## Les quelques briques que l’on a déjà posées

L’équipe, son organisation, tout ça, c’est bien, mais si on parlait un peu plus concret ?

Cette année, **on a décidé de retourner à Toulouse** (souvenez-vous, on y était déjà en [2012](https://sudweb.fr/2012/) et en [2014](https://sudweb.fr/2014/)). On a été tenté pendant un temps de rejouer l’expérience de [l’année dernière à Anduze](https://sudweb.fr/2018/), pour vous ramener en pleine nature, mais ça aurait demandé plus d’organisation et donc cela complexifiait la tâche. Alors qu’avec deux membres de la Thym déjà sur place, il était plus simple de prendre contact et trouver une salle.

Concernant la date, pas de grosse surprise : on a arrêté **les dates du 24 et 25 mai 2019**. On attire quand même votre attention sur le fait que ça tombe en même temps que [MiXiT](https://mixitconf.org/) à Lyon.

> Un dernier élément dont nous avons décidé, c’est qu’il n’y aurait pas d’appel à orateurs cette année. Durant nos différentes réunions, une envie est ressortie de façon assez évidente : et si l’on laissait plus de place aux discussions ? Est-ce qu’on veut vraiment privilégier des moments où 99% des participant·es sont passifs, dans une position d’élèves ? Est-ce que ces personnes n’auraient pas aussi des choses à nous apprendre ? Mais quand même, les conférences c’est aussi l’occasion pour des personnes qui maitrisent un sujet de le partager et de planter des graines de réflexion en chacun de nous. Alors on fait quoi ? Notre choix cette année est de **contacter directement des personnes qu’on aurait envie de voir et de leur laisser un espace d’expression**. L’une des conséquences de ce choix sera un nombre plus faible de conférences, mais aussi plus d’interactions entre les participant·es.

Reviewer: ```suggestion
Un dernier élément dont nous avons décidé, c’est qu’il n’y aurait pas d’appel à orateurs cette année. Durant nos différentes réunions, une envie est ressortie de façon assez évidente : et si l’on laissait plus de place aux discussions ? Est-ce qu’on veut vraiment privilégier des moments où 99&nbsp;% des participant·e·s sont passifs, dans une position d’élèves ? Est-ce que ces personnes n’auraient pas aussi des choses à nous apprendre ? Mais quand même, les conférences c’est aussi l’occasion pour des personnes qui maitrisent un sujet de le partager et de planter des graines de réflexion en chacun de nous. Alors on fait quoi ? Notre choix cette année est de **contacter directement des personnes qu’on aurait envie de voir et de leur laisser un espace d’expression**. L’une des conséquences de ce choix sera un nombre plus faible de conférences, mais aussi plus d’interactions entre les participant·e·s.
```

> La suite des décisions, elle, se dessine sur Slack en ce moment même. Le choix de la salle est quasiment terminé. On discute de comment nous allons nourrir une centaine de bouches de manière peut-être un peu différente de d’habitude. Et puis on réfléchit à comment structurer ces 2 journées de Sud Web pour que tout le monde y trouve son compte.

Reviewer: ```suggestion
La suite des décisions, elle, se dessine sur Slack en ce moment-même. Le choix de la salle est quasiment terminé. On discute de comment nous allons nourrir une centaine de bouches de manière peut-être un peu différente de d’habitude. Et puis on réfléchit à comment structurer ces 2 journées de Sud Web pour que tout le monde y trouve son compte.
```

## Et pour vous, c’est quoi Sud Web ?

> Lors de nos séances de réflexions sur ce que doit être Sud Web, une volonté est revenue sur la table de manière récurrente : **casser les barrières entre organisateurs, conférencières et participant·es**. Ça, ce n’est pas une mince affaire. Alors pour commencer, Thomas a lancé un appel [pour savoir ce que serait le Sud Web de vos rêves](https://twitter.com/oncletom/status/1067509373411438592). Il a déjà eu quelques réponses, mais n’hésitez pas à nous dire si vous voulez aussi participer, on est curieux de savoir ce dont vous rêvez pour la prochaine édition.

Reviewer: ```suggestion
Lors de nos séances de réflexions sur ce que doit être Sud Web, une volonté est revenue sur la table de manière récurrente : **casser les barrières entre organisateurs et organisatrices, intervenant·e·s et participant·e·s**. Ça, ce n’est pas une mince affaire. Alors pour commencer, Thomas a lancé un appel [pour savoir ce que serait le Sud Web de vos rêves](https://twitter.com/oncletom/status/1067509373411438592). Il a déjà eu quelques réponses, mais n’hésitez pas à nous dire si vous voulez aussi participer, on est curieux de savoir ce dont vous rêvez pour la prochaine édition.
```

> You are blocked from following @oncletom and viewing @oncletom's Tweets.

:(
